Koronavilkku/Koronavilkku/Repository/EFGSRepository.swift

```diff
@@ -32,32 +32,24 @@ struct EFGSCountry: Codable, Hashable {
 /// positive COVID-19 diagnosis.
 protocol EFGSRepository {
```

Comment:
MockEFGSRepository does not conform to this protocol anymore

Reply:
🤦